src/katgpucbf/xbgpu/engine.py

```diff
@@ -483,7 +483,7 @@ def _populate_sensors(self) -> None:
     async def _get_in_item(self) -> InQueueItem | None:
         """Get the next :class:`InQueueItem`.
 
-        This is wrapped in a method so that it can be mocked.
+        This is wrapped in a method so it can be mocked.
```

Comment:
Why this change? The grammar read better before (to me).

Reply:
I'd tweaked in when I duplicated the method for each Pipeline. Will roll it back to how it was now-now.